edenai_apis/apis/openai/openai_api.py

```
@@ -24,14 +25,15 @@ class OpenaiApi(
     OpenaiMultimodalApi,
     OpenaiDocParsingApi,
     OpenaiLLMApi,
+    OpenaiVideoApi,
 ):
     provider_name = "openai"
 
     def __init__(self, api_keys: Dict = {}):
```

Ruff (0.13.3)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)